README.md

```diff
@@ -73,6 +73,39 @@ SOLACE_TIMEOUT=5s
 SOLACE_SSL_VERIFY=false
 SOLACE_REDUNDANCY=false</code></pre>
 
+### URL
+
+You can call:
+https://your_exporter:9628/solace-vpn-std?scrapeURI=https%3A%2F%2Fyour-broker%3A943&username=monitoring&password=monitoring
+
+This allows you to over write the parameters:
+- scrapeURI
+- username
+- password
+
+This provides you a single exporter for all your on prem broker.
+
+Security: Only use this feature with HTTPS.
+
+#### Sample prometheus config
+
+<pre><code>- job_name: 'solace-std'
```

Comment:
Side note: it's uncommon to use `<pre><code>` HTML in Markdown to create code blocks - just use triple backticks, that also allows you to specify the language for highlighting: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code